### ISSUE TITLE
[Feature]: Make contact details accessible 🔍✨ 

### ISSUE THREAD

**Issue by 0xabdulkhalid** [Gaurav-Verma07/sampark#400]
## Description
- Currently the Details mentioned on Contact Us section is not accessible by accessibility devices.
- Because everything is just coded with plain text, We need to use well known semantic elements like `Address`, `a` with `mailto` and many more.
- We need to make the contact us form accessible otherwise this will lead some users cannot use our Sampark Application.

## Screenshot (Changes want to proposed to)

![image](https://github.com/Gaurav-Verma07/sampark/assets/92252895/03de7114-8aed-47e9-aeb7-c4cfb5068bce)

**Comment by 0xabdulkhalid** [comment]
Hey there @Gaurav-Verma07 👋🏻 

Please make sure to assign this issue to me to make the work done, Thank You!
